core/worker.cc

```diff
@@ -291,16 +291,16 @@ void *Worker::Run(void *_arg) {
   wid_ = arg->wid;
   core_ = arg->core;
   socket_ = rte_socket_id();
-  DCHECK_GE(socket_, 0);  // shouldn't be SOCKET_ID_ANY (-1)
+  CHECK_GE(socket_, 0);  // shouldn't be SOCKET_ID_ANY (-1)
   fd_event_ = eventfd(0, 0);
-  DCHECK_GE(fd_event_, 0);
+  CHECK_GE(fd_event_, 0);
 
   scheduler_ = arg->scheduler;
 
   current_tsc_ = rdtsc();
 
   packet_pool_ = bess::PacketPool::GetDefaultPool(socket_);
-  DCHECK_NOTNULL(packet_pool_);
```

Comment:
This particular change should perhaps just be folded into the earlier commit in this series (but since there are other DCHECK -> CHECK changes it seems OK to leave it here)